js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -39,7 +56,7 @@ const ConnectedGoogleComboAccountCard = () => {
 			<AccountCard
 				appearance={ APPEARANCE.GOOGLE }
 				alignIcon="top"
-				className="gla-google-combo-account-card--connected"
+				className="gla-google-combo-account-card gla-google-combo-account-card--connected"
```

Comment:
Adding the `gla-google-combo-account-card` to be consistent with the other cards.